lib/resourceutil/util.go

```diff
@@ -49,9 +49,9 @@ func MarkPipelineAsCompleted(logger logr.Logger, obj *unstructured.Unstructured)
 	logger.Info("set conditions", "condition", PipelineCompletedCondition, "value", v1.ConditionTrue)
 }
 
-func PipelineForRequestExists(logger logr.Logger, rr *unstructured.Unstructured, jobs []batchv1.Job) (bool, error) {
+func PipelineExists(logger logr.Logger, obj *unstructured.Unstructured, jobs []batchv1.Job) (*batchv1.Job, error) {
```

Comment:
🤔 i found this renaming a bit confusing
i undestand it's no longer used only for requests, but without the qualifying `ForX`, it no longer conveys the sense that "a pipeline for this spec" exists. unsure how to name it though

Reply:
I gave it a go, lmk what you think